tests/eko/test_matching_n3lo.py

```diff
@@ -143,42 +167,34 @@ def test_Blumlein_3():
             )
 
             np.testing.assert_allclose(
-                aS3[2, 1], ref_val_Hq[L][idx], rtol=2e-5, atol=3e-6
+                aS3[2, 1], ref_val_Hq[L][idx], rtol=2e-5, atol=2e-6
```

Comment:
this is the only *decrease* in accuracy?

Reply:
well Hq was correct also before with the weight 5 harmonics. With the new approximated version we are slightly closer to the known moments, but should be negligible effect.

Reply:
sorry, successfully confused myself ... of course you're also improving here